Doc/README.rst

```diff
@@ -0,0 +1,64 @@
+Contributing to the Biopython Tutorial
+======================================
+
+Installing the tools you need
+-----------------------------
+
+On Ubuntu, you can install these with the following commands::
+
+    sudo apt install hevea
+    sudo apt install texlive-latex-base
+    sudo apt-get install texlive-latex-extra
+    sudo apt-get install python-pygments
```

Comment:
Shouldn't all four lines be ``apt-get``? And can't that be one line?

```
sudo apt-get install hevea texlive-latex-base texlive-latex-extra python-pygments
```

Reply:
After a quick google, I see "apt" has replaced "apt-get" as the preferred command here. Anyway, one line seems clearer :)